radix-engine/src/blueprints/transaction_processor/tx_processor.rs

```diff
@@ -44,11 +44,11 @@ pub struct TransactionProcessorRunInput {
 
 // This needs to match the above, but is easily encodable to avoid cloning from the transaction payload to encode
 #[derive(Debug, Eq, PartialEq, ScryptoEncode)]
-pub struct TransactionProcessorRunInputEfficientEncodable<'a> {
-    pub manifest_encoded_instructions: &'a [u8],
+pub struct TransactionProcessorRunInputEfficientEncodable {
+    pub manifest_encoded_instructions: Rc<Vec<u8>>,
```

Comment:
Not sure if `Rc<[u8]>` as a type makes sense in our case as it's saying: "we need a dynamic lifetime wrapper for an object which has a statically known lifetime"

Reply:
I was just referring to this: https://www.youtube.com/watch?v=A4cKi7PTJSs - an Rc<X> can only access &X, so it makes sense to access &[u8] rather than &Vec<u8>, where the fact it's a Vec is useless.